RandoPlus/NailUpgrades/LogicAdder.cs

```diff
@@ -37,9 +38,9 @@ private static void ApplyLogic(GenerationSettings gs, LogicManagerBuilder lmb)
             if (!RandoPlus.GS.Any) return;
 
             lmb.AddLogicDef(new RawLogicDef(Consts.NailsmithLocationPrefix + "1", $"{SceneNames.Room_nailsmith}[left1]"));
-            lmb.AddLogicDef(new RawLogicDef(Consts.NailsmithLocationPrefix + "2", $"{SceneNames.Room_nailsmith}[left1] + {PALEORE} > 0"));
-            lmb.AddLogicDef(new RawLogicDef(Consts.NailsmithLocationPrefix + "3", $"{SceneNames.Room_nailsmith}[left1] + {PALEORE} > 2"));
-            lmb.AddLogicDef(new RawLogicDef(Consts.NailsmithLocationPrefix + "4", $"{SceneNames.Room_nailsmith}[left1] + {PALEORE} > 5"));
+            lmb.AddLogicDef(new RawLogicDef(Consts.NailsmithLocationPrefix + "2", $"{SceneNames.Room_nailsmith}[left1] + {PALEORE} > 0 + {LISTEN}"));
+            lmb.AddLogicDef(new RawLogicDef(Consts.NailsmithLocationPrefix + "3", $"{SceneNames.Room_nailsmith}[left1] + {PALEORE} > 2 + {LISTEN}"));
+            lmb.AddLogicDef(new RawLogicDef(Consts.NailsmithLocationPrefix + "4", $"{SceneNames.Room_nailsmith}[left1] + {PALEORE} > 5 + {LISTEN}"));
```

Comment:
As a style preference, I think it would look better if the LISTEN fragment appeared the PALEORE fragment

Reply:
You mean like in the constant? Or in the brackets like {LISTEN + PALEORE}?